controllers/app.controllers.js

```diff
@@ -13,6 +13,18 @@ exports.getTopics = (req, res, next) => {
     .catch(next)
 }
 
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params
+    if (isNaN(article_id)) return next({ status: 400, message: 'Not a number, please enter valid id'});
+
```

Comment:
Just as Eli said, you could rely on Psql for this one.

Reply:
done